### PR TITLE
Chapter 11: Account activation

### PR DESCRIPTION
**Pull request by phantuananh1206**
Chapter 11: Account activation

### CODE REVIEW

**Comment by phantuananh1206** [comment]
image

**Comment by phantuananh1206** [comment]
ready

File: app/helpers/account_activations_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (vanvtt-0952) [inline app/helpers/account_activations_helper.rb:1]: Xóa file này nhé

**Comment by vanvtt-0952** [comment]
merged
commented nhỏ, fix rồi mợt

File: app/helpers/account_activations_helper.rb (hunk 1 above)
Comment (phantuananh1206) [reply app/helpers/account_activations_helper.rb:1]: dạ

**Pull request merged**